sda-commons-shared-error/build.gradle

```diff
@@ -4,4 +4,5 @@ dependencies {
 
   testCompile 'junit:junit'
   testCompile 'org.assertj:assertj-core'
+  testRuntimeOnly 'org.junit.vintage:junit-vintage-engine'
```

Comment:
(Why) do we need the dependency?

Reply:
Because of the general Junit5 functionality in build.gradle. It would cause the following error otherwise:
"Cannot create Launcher without at least one TestEngine;"